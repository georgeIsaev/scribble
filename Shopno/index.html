<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Shopno</title>
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<div class="page">
		<header>
			<div class="conteiner">
				<div class="heading row">
					<img src="img/logo.png" alt="" class="logo">
					<nav>
						<div class="menu">
							<div class="menu_icon"></div>
							<div class="menu_list">
								<ul>
									<li><a href="#">Home</a></li>
									<li><a href="#about">About</a></li>
									<li><a href="works">Works</a></li>
									<li><a href="services">Services</a></li>
									<li><a href="team">Team</a></li>
									<li><a href="company">Company</a></li>
									<li><a href="contact">Contact</a></li>
								</ul>
							</div>
						</div>
					</nav>
				</div>
				<div class="title">
					<h1>
						Say hellow to <span class="mark">shopno!</span><br/>
						<span class="mark">Agency</span> corporate shopno theme
					</h1>
					<a href="#" class="buttom">Contact us</a>	
				</div>
			</div>
		</header>
		<section class="about" id="about">
			<div class="conteiner">
				<div class="titles">
					<h2>
						About us	
					</h2>
					<h3>
						Lorem ipsum dolor sit amet event landing template
					</h3>
				</div>
				<div class="about_content row">
					<div class="about_courusel">
						<img src="img/content-img.png" alt="">	
					</div>
					<div class="text">
						<h3>We Actually Do amazing works</h3>
						<p>
							A visual approach is an approach to a runway at an airport<br> 
							conducted under instrument flight rules. The pilot must at<br>  all times have either the airport or the preceding aircraft in <br>  approach correct sight. conducted under instrument flight <br>  rules. The pilot must at all times. <br>
							Preceding aircraft in approach correct sight. conducted   <br>  under instrument flight rules.
						</p>
						<h3>Why we are the great?</h3>
						<p>
							A visual approach is an approach to a runway at an airport 
							conducted under instrument flight rules.<br>
							The preceding aircraft in approach correct sight.
						</p>
					</div>
				</div>
			</div>
		</section>
		<section class="works">
			<div class="conteiner">
				<div class="titles">
					<h2>
						Featured works	
					</h2>
					<h3>
						Lorem ipsum dolor sit amet event landing template
					</h3>
				</div>
				<div class="works_img">
					<div class="works_first row">
						<img src="img/work-1.jpg" alt="">
						<img src="img/work-2.jpg" alt="">
						<img src="img/work-3.jpg" alt="">
					</div>	
					<div class="works_second row">
						<img src="img/work-4.jpg" alt="">
						<img src="img/work-5.jpg" alt="">
						<img src="img/work-6.jpg" alt="">
					</div>
				</div>
			</div>	
		</section>
		<section class="services">
			<div class="conteiner">
				<div class="titles">
					<h2>
						Our services	
					</h2>
					<h3>
						Lorem ipsum dolor sit amet event landing template
					</h3>
				</div>
				<div class="services_content">
					<div class="first_row row">
						<div class="first_block">
							<h3>
								Designing is The CooL LOVE FOR.
							</h3>
							<p>
								Our most popular service is our Virtual Receptionist. We know that sometimes it’s something and you middle most this job was posted less than five minutes ago, and we think it’s a good match for you. If you submit a proposal now, you’ll be one of the first candidates. You may even catch the client while they’re still online.
							</p>
							<a href="#">
								<img src="img/service-icon.png" alt="">	
							</a>
						</div>
						<div class="second_block">
							<div class="second_block_hover small_block">
								<h3>
									Cool psd Design	
								</h3>
								<p>
									Our most popular service is our we know that sometimes it’s something and you middle most..	
								</p>
								<a href="#" class="block_href">
									<span>Know More</span>
									<img src="img/service-icon-2.png" alt="">
								</a>
							</div>	
						</div>
					</div>
					<div class="second_row row">
						<div class="third_block">
							<div class="third_block_hover big_block">
								<h3>
									Designing is The Cool	
								</h3>
								<p>
									Our most popular service is our Virtual Receptionist. We know that ...	
								</p>
								<a href="#" class="block_href">
									<span>Know More</span>
									<img src="img/service-icon-2.png" alt="">
								</a>
							</div>
						</div>
						<div class="fourth_block">
							<div class="fourth_block_hover small_block">
								<h3>
									PSD TO HTML KORI	
								</h3>
								<p>
									We know that sometimes it’s something and you middle most..	
								</p>
								<a href="#" class="block_href">
									<span>Know More</span>
									<img src="img/service-icon-2.png" alt="">
								</a>
							</div>	
						</div>	
					</div>
					<div class="third_row row">
						<div class="fifth_block">
							<div class="fifth_block_hover small_block">
								<h3>
									Cool psd Design	
								</h3>
								<p>
									Our most popular service is our new virtual Receptionist you middle most..
								</p>
								<a href="#" class="block_href">
									<span>Know More</span>
									<img src="img/service-icon-2.png" alt="">
								</a>
							</div>	
						</div>
						<div class="sixth_block">
							<div class="sixth_block_hover big_block">
								<h3>
									Designing is The Cool FOR
								</h3>
								<p>
									Our most popular service is our Virtual Receptionist. We know that sometimes it’s something and you middle most..
								</p>
								<a href="#" class="block_href">
									<span>Know More</span>
									<img src="img/service-icon-2.png" alt="">
								</a>
							</div>		
						</div>	
					</div>	
				</div>	
			</div>
		</section>
		<section class="team">
			<div class="conteiner">
				<div class="titles">
					<h2>
						Featured works	
					</h2>
					<h3>
						Lorem ipsum dolor sit amet event landing template
					</h3>
				</div>
				<div class="team_img">
					<div class="team_first row">
						<img src="img/team-img-1.jpg" alt="">
						<img src="img/team-img-2.jpg" alt="">
						<img src="img/team-img-3.jpg" alt="">
					</div>	
					<div class="team_second row">
						<img src="img/team-img-4.jpg" alt="">
						<img src="img/team-img-5.jpg" alt="">
						<img src="img/team-img-6.jpg" alt="">
					</div>
				</div>
			</div>
		</section>
		<section class="says">
			<div class="conteiner">
				<div class="titles">
					<h2>
						What they says	
					</h2>
					<h3>
						Lorem ipsum dolor sit amet event landing template
					</h3>
				</div>
				<div class="says_content row">
					<div class="about_courusel">
						<img src="img/says-img.png" alt="">	
					</div>
					<div class="says_text">
						<h3>AL RAYHAN <em>/ UI/UX DESIGNER</em></h3>
						<p>
							Lorem Ipsum is simply dummy text of the printing and you 
							standard dummy know that text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
						</p>
						<img src="img/stars.png" alt="">	
					</div>
					<div class="courusel_buttom">
						<div class="last">
							<a href="#">
								<img src="img/icon-last.png" alt="">
							</a>	
						</div>
						<div class="next">
							<a href="#">
								<img src="img/icon-next.png" alt="">
							</a>	
						</div>
					</div>
				</div>

			</div>
		</section>
		<section class="company">
			<div class="conteiner">
				<img src="img/back-logo.jpg" alt="">
			</div>
		</section>
		<section class="contact">
			<div class="conteiner">
				<div class="contact_form row">
					<div class="first_form"></div>
					<div class="second_form"></div>
				</div>
			</div>
		</section>
		<footer></footer>
	</div>	
</body>
</html>