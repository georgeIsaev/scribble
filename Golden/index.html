<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<div class="page">

		<header>
			<div class="conteiner">
				<div class="heading clearfix">
					<img src="img/logo.png" alt="" class="logo">
					<nav>
						<ul class="menu">
							<li>
								<a href="#">Home</a>
							</li>
							<li>
								<a href="#">Services</a>
							</li>
							<li>
								<a href="#">Portfolio</a>
							</li>
							<li>
								<a href="#">About</a>
							</li>
							<li>
								<a href="#">Contact</a>
							</li>
						</ul>
					</nav>
				</div>
				<div class="titels">
					<div class="titels_first">
						Welcome To Our Studio!
					</div>
					<h1>
						It’s nice to Meet you
					</h1>
					<a href="#" class="button">Tell Me More</a>
				</div>
			</div>
		</header>

		<section id="services">
			<div class="conteiner">
				<div class="title">
					<h2>Services</h2>
					<p>
						Proin iaculis purus consequat sem cure. 
					</p>
				</div>
				<div class="services clearfix">
					<div class="services_item">
						<img src="img/icon1.png" alt="">
						<h3>E-Commerce</h3>
						<p>
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
					<div class="services_item">
						<img src="img/icon2.png" alt="">
						<h3>Responsive Web</h3>
						<p>
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
					<div class="services_item">
						<img src="img/icon3.png" alt="">
						<h3>Web Security</h3>
						<p>
							Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit 
							aenean rhoncus posuere odio in tincidunt.
						</p>
					</div>
				</div>
			</div>
		</section>
		<section id="portfolio">
			<div class="conteiner">
				<div class="title">
					<h2>Our portfolio</h2>
					<p>
						Proin iaculis purus consequat sem cure.  
					</p>
				</div>
				<div class="works clearfix">
					<img src="img/img1.jpg" alt="">
					<img src="img/img2.jpg" alt="">
					<img src="img/img3.jpg" alt="">
				</div>
			</div>
		</section>
		<section>
			<div class="conteiner">
				<div class="title">
					<h2>About Us</h2>
					<p>
						Proin iaculis purus consequat sem cure.
					</p>
				</div>
				<div class="about">
					<div class="about_item clearfix">
						<div class="about_title">
							<h3>July 2010<br> Our Humble Beginnings</h3>
							<p>
								Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit.
							</p>
						</div>
						<img src="img/about1.jpg" alt="">
					</div>
					<div class="about_item clearfix">
						<img src="img/about2.jpg" alt="">
						<div class="about_title">
							<h3>January 2011<br> Facing Startup Battles</h3>
							<p>
								Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean 
								rhoncus posuere odio in tincidunt.
							</p>
						</div>
					</div>
					<div class="about_item clearfix">
						<div class="about_title">
							<h3>December 2012<br> Enter The Dark Days</h3>
							<p>
								Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni.
							</p>
						</div>
						<img src="img/about3.jpg" alt="">
					</div>
					<div class="about_item clearfix">
						<img src="img/about4.jpg" alt="">
						<div class="about_title">
							<h3>February 2014<br> Our Triumph</h3>
							<p>
								Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean.
							</p>
						</div>
					</div>
				</div>
			</div>
		</section>
		<footer>
			<div class="conteiner">
				<div class="footer">
					<p>
						© Copyright 2014 FreebiesXpress.com
					</p>
				</div>
			</div>
		</footer>
	</div>
</body>
</html>